test/e2e/connectivity_test.go

```diff
@@ -114,7 +114,7 @@ func (data *TestData) testHostPortPodConnectivity(t *testing.T) {
 		t.Fatalf("Error when creating busybox test Pod: %v", err)
 	}
 	defer deletePodWrapper(t, data, clientName)
-	if _, err := data.podWaitForIP(defaultTimeout, clientName); err != nil {
+	if _, err := data.podWaitForIP(defaultTimeout, testNamespace, clientName); err != nil {
```

Comment:
I think namespace is the third argument? Could the test pass?

Reply:
Thanks, I'm too careless. Updated.